### PR TITLE
Implements the view and controller of the members list

### PR DESCRIPTION
**Pull request by romain-pierre**

### CODE REVIEW

File: app/Resources/views/committee/_sidebar.html.twig [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+                        <a href="mailto:{{ host.emailaddress }}">Contacter</a>
+                    {% endif %}
+                </li>
+            {% endfor %}
+        </ul>
+    </div>
+    <div class="committee-details">
+        <h5>Localisation</h5>
+        {#
+            TODO add static image
```
Comment (tgalopin) [inline app/Resources/views/committee/_sidebar.html.twig]: Didn't we merge the static map?

File: app/Resources/views/committee/_sidebar.html.twig [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    <div class="committee-details">
+        <a href="#">Signaler ce comité local</a>
+    </div>
+    <div class="committee-details">
+        <h4>{{ 'committee.members_count'|transchoice(committee_members_count) }}</h4>
+        <h5>À propos</h5>
+        <p id="committee-description">{{ committee.description }}</p>
+    </div>
+    <div class="committee-details">
+        <h5>Administration</h5>
```
Comment (tgalopin) [inline app/Resources/views/committee/_sidebar.html.twig]: This block should be displayed only when animateur, is it for another PR?

File: app/Resources/views/committee/members.html.twig [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+{% block content %}
+    {% set is_host = is_host(committee) %}
+
+    <section class="l__wrapper--medium">
+        <h1>{{ committee.name }}</h1>
+
+        <a href="#">Discussion</a>
+        <strong>Membres</strong>
+
+        <a href="#">Editer le comité</a>
```
Comment (hhamon) [inline app/Resources/views/committee/members.html.twig]: Can you add the link for this one please. The route was merged this morning.

File: app/Resources/views/committee/members.html.twig [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+{% block content %}
+    {% set is_host = is_host(committee) %}
+
+    <section class="l__wrapper--medium">
+        <h1>{{ committee.name }}</h1>
+
+        <a href="#">Discussion</a>
+        <strong>Membres</strong>
+
+        <a href="{{ path('app_committee_edit', { uuid: committee.uuid, slug: committee.slug } ) }}">Editer le comité</a>
```
Comment (hhamon) [inline app/Resources/views/committee/members.html.twig]: Use `{{ committee_path('app_committee_edit', committee) }}` instead and make sure this link is visible only for committee hosts.

**Pull request merged**